src/types/rule.d.ts

```diff
@@ -155,10 +155,6 @@ export type MQTTIn = MQTTOut & {
   local_qos: QoSLevel
 }
 
-export type OtherBridge = Record<string, any>
-
-export type BridgeItem = HTTPBridge | MQTTOut | MQTTIn | OtherBridge
```

Comment:
Why has otherBridge been removed here, is it meant to say that this type only exists on the enterprise branch?

Reply:
I change back.